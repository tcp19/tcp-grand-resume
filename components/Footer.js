import {
    Box,
    Container,
    Stack,
    Text,
    useColorModeValue,
} from "@chakra-ui/react";

import React from "react";

export const Footer = () => (
    <Box
        bg={useColorModeValue("gray.50", "gray.900")}
        color={useColorModeValue("gray.700", "gray.200")}
    >
        <Container
            as={Stack}
            maxW={"6xl"}
            py={4}
            direction={{ base: "column", md: "row" }}
            spacing={4}
            justify={{ base: "center", md: "space-between" }}
            align={{ base: "center", md: "center" }}
        >
            <Stack direction={"row"} spacing={6}>
                <Text>Build with ❤️ from Nigeria. Teencode Africa </Text>
            </Stack>
            <Text>© {new Date().getFullYear()}. All rights reserved</Text>
        </Container>
    </Box>
);

export default Footer;
