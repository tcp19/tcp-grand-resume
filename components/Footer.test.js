import { render } from "@testing-library/react";
import Footer from "./Footer";

describe("Footer", () => {
    it("should render properly", () => {
        const { container } = render(<Footer />);
        expect(container).toMatchSnapshot();
    });
});
