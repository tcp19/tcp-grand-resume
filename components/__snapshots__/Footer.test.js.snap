// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render properly 1`] = `
<div>
  <div
    class="css-1sp5dw5"
  >
    <div
      class="chakra-stack chakra-container css-1pyonn0"
    >
      <div
        class="chakra-stack css-111rh5x"
      >
        <p
          class="chakra-text css-0"
        >
          Build with ❤️ from Nigeria. Teencode Africa 
        </p>
      </div>
      <p
        class="chakra-text css-0"
      >
        © 
        2021
        . All rights reserved
      </p>
    </div>
  </div>
</div>
`;
